pkg/controller/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -265,19 +284,193 @@ func TestReconcile(t *testing.T) {
 		}
 		result, err := r.Reconcile(request)
 		if result != tc.expected.result {
-			t.Errorf("Expected %v, got: %v", tc.expected.result, result)
+			t.Errorf("Test case : %v. Expected: %v, got: %v", tc.node.Name, tc.expected.result, result)
```

Comment:
nit: `"Test case:` (i.e., no space) like the formatting in `TestGetNodeCondition`.